src/app/inbox/email-create/email-create.component.html

```diff
@@ -0,0 +1,8 @@
+<button (click)="showModal = true" class="ui button inverted primary fluid">
+  Compose
+</button>
+
+<app-modal (dismiss)="showModal = false" *ngIf="showModal">
```

Comment:
What is `showModal = false` being used for here? If it's checking for false value `!showModal` or `showModal === false` is the right syntax.